teachbooks/external_content/licenses.py

```diff
@@ -39,6 +39,8 @@ def find_license(repo_toplevel: str | Path) -> str | None:
         license_file = Path(repo_toplevel) / "LICENSE"
     elif (Path(repo_toplevel) / "LICENSE.md").exists():
         license_file = Path(repo_toplevel) / "LICENSE.md"
+    elif (Path(repo_toplevel) / "LICENSE.txt").exists():
+        license_file = Path(repo_toplevel) / "LICENSE.txt"
     else:
```

Comment:
Apparently github also checks .rst files, would be good to add just in case; https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#determining-the-location-of-your-license

At this point having four if statements does make me itch a bit. I think this should work:

```py
valid_extensions = ("", ".md", ".txt", ".rst")
for ext in valid_extensions:
    license_file = Path(repo_toplevel) / f"LICENSE{ext}"
    if license_file.exists():
        break
else:
    <raise error>
```

(note: the "else" statement after a for loop only runs if the for loop has reached the final element without being broken out of)